.gitignore

```diff
@@ -18,3 +18,7 @@
 
 /tools/gnmsg/.idea/
 /tools/gnmsg/__pycache__/
+
+/netcore/*/bin/
```

Comment:
Is there any way to get .NET build tools to not build in the source tree?

Reply:
Also, can you put these in a `.gitignore` in the `netcore` directory.

Reply:
> Is there any way to get .NET build tools to not build in the source tree?

Not as far as I know.  A quick web search yields a lot of info about MSBuild-style csproj files, but nothing about the new improved dotnet kind.

Reply:
Ok, so this is a function you you checking in the csproj as source rather than generating it with CMake.

Reply:
RIght, cmake generates the "old" style .csproj and .sln files.

Reply:
> Also, can you put these in a `.gitignore` in the `netcore` directory.

Done.